tang/Sanity/simple-tang-server-start/main.fmf

```diff
@@ -1,24 +1,24 @@
 summary: Run tang server, which stays running after test finishes. For manual testing
     with --reserve parameter.
-description: ''
-contact: Martin Zelený <mzeleny@redhat.com>
+component+:
```

Comment:
Is this "component+" a new feature, or is it a typo? If not a typo ... what is it used for?

Reply:
This is the feature. It extends the inherited list.

Reply:
OK. Sorry for my ignorance. Thanks for clarifying